src/jupyter_spark/static/jupyterlab-plugin.js

```diff
@@ -0,0 +1,54 @@
+import "./spark.css";
+
+import "bootstrap/dist/css/bootstrap.min.css";
+
+import {
+    JupyterLab, JupyterLabPlugin
+} from '@jupyterlab/application';
+
+import {
+    NotebookActions, NotebookPanel, INotebookModel
+} from '@jupyterlab/notebook';
+
+import {
+    ToolbarButton
+} from '@jupyterlab/apputils';
+
+import {
+    IDisposable, DisposableDelegate
+} from '@phosphor/disposable';
+
+import {
+    Widget, PanelLayout
+} from '@phosphor/widgets';
+
+var common = require('./extension.js');
+
+const plugin = {
+    id: 'jupyter_spark',
+    autoStart: true,
+    activate: (app) => {
+        let api_url = "/spark/api/v1";
```

Comment:
In the past we had trouble with the base path for the API, are you sure that's not the case here?

Reply:
Thanks for the reminder.  Indeed it looks like we have the same problem here.  I'll fix.

Reply:
Should be fixed now.